frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -1,36 +1,49 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useActions, useValues } from 'kea'
 import dayjs from 'dayjs'
 import { trendsLogic } from 'scenes/trends/trendsLogic'
 import { DownloadOutlined } from '@ant-design/icons'
-import { Modal, Button, Spin } from 'antd'
+import { Modal, Button, Spin, Input } from 'antd'
 import { PersonsTable } from 'scenes/persons/PersonsTable'
-import { Link } from 'lib/components/Link'
-import { ArrowRightOutlined, ClockCircleOutlined } from '@ant-design/icons'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 import { ViewType } from 'scenes/insights/insightLogic'
 import { toParams } from 'lib/utils'
+import { PersonType } from '~/types'
+import Fuse from 'fuse.js'
 
 interface Props {
     visible: boolean
     view: ViewType
     onSaveCohort: () => void
 }
 
+const searchPersons = (sources: PersonType[], search: string): PersonType[] => {
+    return new Fuse(sources, {
```

Comment:
we only get the first 100 people on pagination so this won't work. You'll need to change the people endpoint in the backend (can limit to just Clickhouse to cut down on scope).

Reply:
can you clarify which part of it won't work? i'm a little confused 😅 
(i'm searching and pressing enter for the result here)

https://user-images.githubusercontent.com/25164963/122967641-ad1eed00-d358-11eb-97e6-025a57816cfd.mov

Reply:
If you click on a data point that has more than 100 persons, it will only search the emails of the first 100 users

Reply:
ahh i see now, the peoples endpoint exists under actions so I wanted to avoid manually writing a search filter for it and passing in more params, but it'll need to be done. Unless there's a better way?